src/models/db/migrations/20240628090906_query_table.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  const now = () => knex.fn.now()
+
+  await knex.schema.createTable('queries', (def) => {
```

Comment:
We currently use singulars as table names for example `connection` so this should be `query`